google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeployAction.java

```diff
@@ -171,6 +171,9 @@ public void processTerminated(final ProcessEvent event) {
       try {
         if (event.getExitCode() == 0) {
           callback.succeeded(new DeploymentRuntimeImpl(deploymentOutput.toString(), version));
+        } else if (event.getExitCode() == 143) {
```

Comment:
It looks like the exit code on Windows is always 1 when the process terminates. So, we can't rely on the exit code being anything other than 0 or >0.